### PR TITLE
Payment backend

### PR DESCRIPTION
**Pull request by MangeshThakre**

### CODE REVIEW

File: backend/controller/payment.controller.js [hunk 1]
```diff
@@ -179,8 +164,8 @@ const verifySubscription = asyncHandler(async (req, res, next) => {
 
 const createPlan = asyncHandler(async (req, res, next) => {
   const { planName, amount, description, active } = req.body;
-
-  if (!planName || !amount || !description || !active) {
+  console.log(req.body);
```
Comment (shivamvijaywargi) [inline backend/controller/payment.controller.js:167]: Remove console log

File: backend/controller/payment.controller.js [hunk 2]
```diff
@@ -179,8 +164,8 @@ const verifySubscription = asyncHandler(async (req, res, next) => {
...
+  if (!planName || !amount || !description) {
     return next(
       new CustomError(
         "All fields are required. planName, amount, description, active"
```
Comment (shivamvijaywargi) [inline backend/controller/payment.controller.js:171]: You are not checking active, then why send it in response?

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please check the comments

File: backend/controller/payment.controller.js (hunk 1 above)
Comment (MangeshThakre) [reply backend/controller/payment.controller.js:167]: done

File: backend/controller/payment.controller.js (hunk 2 above)
Comment (MangeshThakre) [reply backend/controller/payment.controller.js:171]: removed

**Review by shivamvijaywargi — APPROVED** [review]

**Pull request merged**